integration/localstack/docker-compose.yml

```diff
@@ -3,7 +3,9 @@ version: "3.8"
 services:
   localstack:
     container_name: "${LOCALSTACK_DOCKER_NAME-localstack_main}"
-    image: localstack/localstack
+    # @TODO use latest when this is fixed https://github.com/localstack/localstack/issues/5502
+    # Use 0.12.20 since this is last version that worked for now
+    image: localstack/localstack:0.12.20
```

Comment:
With this test going from a handful of instances to a wide assortment of instances, is there opportunity for localstack to run in a central location instead of on every instance where the test is running?  Running in a mode of many integration tests to one localstack?

Reply:
It is a bit complicated setting this up I think. I can create a ticket for that and take care of it in a new pr.